src/Components/Server/src/BlazorPack/BlazorPackHubProtocolWorker.cs

```diff
@@ -34,6 +35,16 @@ protected override object DeserializeObject(ref MessagePackReader reader, Type t
                 {
                     return reader.ReadSingle();
                 }
+                else if (type == typeof(byte[][]))
+                {
+                    var length = reader.ReadArrayHeader();
+                    var result = new byte[length][];
```

Comment:
FYI this is bad, length comes from user provided input, so you cannot pre-allocate the array.

Reply:
Good call.

Reply:
Thanks, updated to utilize a list for the result.